script/openqa-clone-custom-git-refspec

```diff
@@ -135,7 +135,9 @@ Please try 'curl $json_url' or select another job, e.g. in the same scenario: $h
     local casedir="${casedir:-"$repo#$branch"}"
     local GROUP="${GROUP:-0}"
     local dry_run="${dry_run:-""}"
-    local cmd="$dry_run openqa-clone-job $clone_args \"$host\" \"$job\" _GROUP=\"$GROUP\" TEST=\"$test\" BUILD=\"$build\" CASEDIR=\"$casedir\" PRODUCTDIR=\"$productdir\" NEEDLES_DIR=\"$needles_dir\""
+    local scriptdir
+    scriptdir=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
I am not familiar with shell script, just a  small question: why not write `local script=$(dirname "${BASH_SOURCE[0]}")` directly?

Reply:
Likely shellcheck didn't like it.

Reply:
Yes. Supposedly it's better to do it separately because it could hide problems otherwise.